src/sortStudents.ts

```diff
@@ -1,16 +1,56 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+export type SortOrder = 'asc' | 'desc';
+
+export function getAverageGrade(grade: number[]): number {
+  return grade
+    .reduce((prev: number, el: number) => (prev + el), 0) / grade.length;
 }
 
-// create SortOrder type
-export type SortOrder;
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const stArr: Student[] = [...students];
+
+  stArr.sort((a: Student, b: Student) => {
```

Comment:
```suggestion
 [...students].sort((a: Student, b: Student) => {
```
you can do copy here instead of creating variable